syntax = "proto3";

package googlemap;

service ValidateAddressService {
  rpc ValidateAddress(ValidateAddressRequest) returns (ValidateAddressResponse);
}

message ValidateAddressRequest {
  string address = 1;
}

message ValidateAddressResponse {
  bool is_valid = 1;
  string validated_address = 2;
}
